MacPass/MPPasswordInputController.m

```diff
@@ -77,6 +77,11 @@ - (void)viewDidLoad {
   [self.togglePasswordButton bind:NSEnabledBinding toObject:self withKeyPath:NSStringFromSelector(@selector(enablePassword)) options:nil];
   [self.passwordTextField bind:NSEnabledBinding toObject:self withKeyPath:NSStringFromSelector(@selector(enablePassword)) options:nil];
   [self _reset];
+  
+  touchBarIdentifier = @"com.hicknhacksoftware.MacPass.TouchBar";
```

Comment:
Initalize them statically so no need to dynamically allocate them every time the view get's loaded (e.g. if we use more than one document the init isn't needed every time and the storage doesn't need to reside for every object)